e2e/node/src/node-server.test.ts

```diff
@@ -14,11 +14,16 @@ import {
 } from '@nx/e2e/utils';
 import { join } from 'path';
 
-xdescribe('Node Applications + webpack', () => {
+describe('Node Applications + webpack', () => {
```

Comment:
There's another `xdescribe` in this file. Could you enable that as well please?

Reply:
there are no `xdescribe`s in this file you can see the output here: https://staging.nx.app/runs/oaKPGjBQVJ?utm_source=pull-request&utm_medium=comment